slobrok/src/vespa/slobrok/server/exchange_manager.cpp

```diff
@@ -112,9 +121,16 @@ ExchangeManager::healthCheck()
             if (! diff.empty()) {
                 LOG(warning, "Diff from consensus map to peer slobrok mirror: %s",
                     diff.c_str());
+                someBad = true;
             }
         }
     }
+    if (someBad) {
+        _env.setConsensusTime(now() - _lastFullConsensusTime);
```

Comment:
You could also utilize the time utilities in vespalib/util/time.h by @havardpe 
vespalib:.steady_time _lastFullConsensusTime;
_env.setConsensusTime(vespalib::toS(steady_clock::now() - _lastFullConsensusTime))